src/interfaces/module.ts

```diff
@@ -0,0 +1,9 @@
+export default interface Module {
```

Comment:
@Guergeiro , in your google docs diagram you named interfaces ISomeName. Should we do that here like `IModule`?

Reply:
personally i like the `IModule`

Reply:
I personally prefer that approach, makes it more explicit to what's an interface and what isn't